l10n_it_corrispettivi/wizards/account_invoice_refund.py

```diff
@@ -0,0 +1,29 @@
+#  -*- coding: utf-8 -*-
+#  Copyright 2019 Simone Rubino - Agile Business Group
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class AccountInvoiceRefund(models.TransientModel):
+    _inherit = "account.invoice.refund"
+
+    @api.multi
+    def compute_refund(self, mode='refund'):
+        res = super(AccountInvoiceRefund, self).compute_refund(mode=mode)
+        if isinstance(res, dict):
+            invoice_ids_domain = res['domain'][-1]
+            invoices = self.env['account.invoice'].search([invoice_ids_domain])
+            if all(not inv.corrispettivo for inv in invoices):
+                return res
+
+            # Some of the created refunds are corrispettivi:
```

Comment:
Possono esistere situazioni miste? In cui `invoices` contiene sia fatture che ricevute?

Reply:
@eLBati in teoria sì, nel caso in cui il wizard (o anche solo `compute_refund`) venisse chiamato da una voce del menu 'azioni' su più fatture miste con ricevute.
In pratica no perché l'azione del wizard `action_account_invoice_refund` (in Odoo core) è solo associata ad un bottone sulla fattura che quindi non credo possa dare risultati misti.